### PR TITLE
[RUN-9312]add projv2 to RN and setup page

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

File: docs/admin/admin-ui-setup/project-setup.md [hunk 1]
```diff
@@ -46,10 +46,24 @@ As an administrator, you may want to disconnect the two parameters. So, for exam
 !!! Note
     To be able to create or edit Projects, you must have *Editor* access. See the [Users](admin-ui-users.md) documentation.
 
-* Login to the Projects area of the Run:ai user interface at `<company-name>.run.ai`.
-* On the top right, select "Add New Project"
-* Choose a Project name and a Project quota
-* Press "Save"
+1. In the left-menu, press **Projects**.
+2. Choose a *Department* from the drop-down. The default is `default`.
```
Comment (Hagay-RunAI) [inline docs/admin/admin-ui-setup/project-setup.md:50]:
```suggestion
1.5 On the top right, select "Add New Project"
2. Choose a *Department* from the drop-down. The default is `default`.
```

File: docs/admin/admin-ui-setup/project-setup.md [hunk 2]
```diff
@@ -46,10 +46,24 @@ As an administrator, you may want to disconnect the two parameters. So, for exam
...
+3. Enter a *Project name*. Press *Namespace* to set the namespace associated with the project. You can either create the namespace from the project name (default) or enter an existing namespace.
+4. In *Access control*, add one or more applications or users. If your user or application isn't in the list, see [Roles and permissions](admin-ui-users.md#roles-and-permissions), and verify that the users have the correct permissions. To change user permissions, see [Working with users](admin-ui-users.md#working-with-users).
+5. In *Quota management*, configure the node pool priority (if editable), the GPUs, CPUs, CPU memory, and Over quota priority settings. Configure the following:
+
+       * *Order of priority*&mdash;the priority the node pool will receive when trying to schedule workloads. For more information, see [Node pool priority](../../Researcher/scheduling/using-node-pools.md#multiple-node-pools-selection).
```
Comment (Hagay-RunAI) [inline docs/admin/admin-ui-setup/project-setup.md:55]: Order of priority - the priority the node-pool will receive when trying to schedule a workload and the workload did not specify its own list of priority for node-pools, or a namespace policy dictated the order of priority.

Comment (jasonnovichRunAI) [reply docs/admin/admin-ui-setup/project-setup.md:56]: Did the link explain the order of priority correctly? Does it need to be changed there? Generally in a procedure I don't go into explanations, I try to reference the pages where it was already explained (if there is one). Can you review the referenced link?

Comment (Hagay-RunAI) [reply docs/admin/admin-ui-setup/project-setup.md:56]: ok

**Review by Hagay-RunAI — APPROVED** [review]

**Pull request merged**